### PR TITLE
chore(release): prepare v0.0.9

### PR DESCRIPTION
**Pull request by dergigi**
Prepares the v0.0.9 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [0.0.9](https://github.com/joinmarket-webui/joinmarket-webui/compare/v0.0.8...v0.0.9) (2022-07-14)
+
+#### Fixed
+
+* advanced wording and behavior ([#390](https://github.com/joinmarket-webui/joinmarket-webui/issues/390)) ([c25c3ce](https://github.com/joinmarket-webui/joinmarket-webui/commit/c25c3ce6259112c702e249e4af3396001f8bbe3e))
+* change frozen icon ([#379](https://github.com/joinmarket-webui/joinmarket-webui/issues/379)) ([b4590c9](https://github.com/joinmarket-webui/joinmarket-webui/commit/b4590c9be1be0edbbcf6bfda5030876b2a7b64fb))
```
Comment (dnlggr) [inline CHANGELOG.md:13]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+* disable 'create wallet' link when unlocking wallet ([#334](https://github.com/joinmarket-webui/joinmarket-webui/issues/334)) ([e3083b9](https://github.com/joinmarket-webui/joinmarket-webui/commit/e3083b9c8ba5a99747f588188769b954ed03f0c2))
+* do not show FB create form when maker is running ([#384](https://github.com/joinmarket-webui/joinmarket-webui/issues/384)) ([e0f51fa](https://github.com/joinmarket-webui/joinmarket-webui/commit/e0f51faacbef44a3914c3f000722761362b47ee0))
+* don not show expired fidelity bonds as locked ([#378](https://github.com/joinmarket-webui/joinmarket-webui/issues/378)) ([0f7d590](https://github.com/joinmarket-webui/joinmarket-webui/commit/0f7d590e352d0dd1f8dce6e55a495b789aef2d31))
```
Comment (dnlggr) [inline CHANGELOG.md:16]:
```suggestion
* do not show expired fidelity bonds as locked ([#378](https://github.com/joinmarket-webui/joinmarket-webui/issues/378)) ([0f7d590](https://github.com/joinmarket-webui/joinmarket-webui/commit/0f7d590e352d0dd1f8dce6e55a495b789aef2d31))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+* pass body of confirm modal via child node ([#377](https://github.com/joinmarket-webui/joinmarket-webui/issues/377)) ([23f7383](https://github.com/joinmarket-webui/joinmarket-webui/commit/23f738355153df8f88164584bbde236426e5d30e))
+* reload utxos after creating a fidelity bond ([#380](https://github.com/joinmarket-webui/joinmarket-webui/issues/380)) ([72a3e8c](https://github.com/joinmarket-webui/joinmarket-webui/commit/72a3e8cff6bfa2cc6bbbd3f6afcefe51f4830d15))
+
+#### Added
+
+* batch unfreeze UTXOs after creating fidelity bond ([#388](https://github.com/joinmarket-webui/joinmarket-webui/issues/388)) ([efa3361](https://github.com/joinmarket-webui/joinmarket-webui/commit/efa336117779b6f24a5165b41c9e863c85b90c7c))
+* mention jar [#0](https://github.com/joinmarket-webui/joinmarket-webui/issues/0) in preconditions ([#395](https://github.com/joinmarket-webui/joinmarket-webui/issues/395)) ([08b3e7f](https://github.com/joinmarket-webui/joinmarket-webui/commit/08b3e7f4b15151e406920e04eec5c35a154ee7c2))
```
Comment (dnlggr) [inline CHANGELOG.md:25]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+* move fidelity bonds to earn screen ([#361](https://github.com/joinmarket-webui/joinmarket-webui/issues/361)) ([8608329](https://github.com/joinmarket-webui/joinmarket-webui/commit/8608329f3e45f1cc6c5cae43f8960ae01a9e147c))
+* time-lock-icon ([#402](https://github.com/joinmarket-webui/joinmarket-webui/issues/402)) ([c361837](https://github.com/joinmarket-webui/joinmarket-webui/commit/c361837fd52100f36f61b13a00ddb935d550c4ce))
```
Comment (dnlggr) [inline CHANGELOG.md:27]:
```suggestion
```

**Review by dnlggr — APPROVED** [review]
LGTM ✅

Might want to remove a couple of irrelevant changelog entries but not really important.

**Review by theborakompanioni — APPROVED** [review]

**Pull request merged**